src/clj/game/cards/ice.clj

```diff
@@ -2104,6 +2104,38 @@
                    :async true
                    :effect (req (do-access state :runner eid [:rd] {:no-root true}))}]}
 
+   "Slot Machine"
+   {:implementation "Encounter effect is manual"
+    :abilities [{:label "Roll them bones"
+                 :effect (effect (move :runner (first (:deck runner)) :deck)
+                                 (system-msg (str "uses Slot Machine to put the top card of the stack to the bottom,"
+                                                  " then reveal the top 3 cards in the stack: "
+                                                  (join ", " (map :title (take 3 (:deck runner)))))))}]
```

Comment:
I wonder if we want to help the mental math and print the types of each card here as well. E.g. "Sure Gamble (event), Career Fair (event), Bankroll (program)".

Reply:
Very true! I'll do that.